local-cli/generator-common/index.js

```diff
@@ -1,16 +1,35 @@
+// @ts-check
+
 const fs = require('fs');
-const chalk = require('chalk');
+const chalk = require('chalk').default;
 const path = require('path');
 const copyAndReplace = require('@react-native-community/cli/build/tools/copyAndReplace').default;
 const walk = require('@react-native-community/cli/build/tools/walk').default;
 const prompt = require('@react-native-community/cli/build/tools/generator/promptSync').default();
 
+/**
+ * @param {string} destPath
+ */
 function createDir(destPath) {
   if (!fs.existsSync(destPath)) {
-    fs.mkdirSync(destPath);
+    fs.mkdirSync(destPath, { recursive: true });
   }
 }
 
+/**
+ * @todo Move this upstream to @react-native-community/cli
```

Comment:
How are we tracking this work?

Reply:
I’ll be touching the CLI package for the next iteration of this when I make auto-linking work and will make this change then.